Packs/FraudWatch/pack_metadata.json

```diff
@@ -12,5 +12,14 @@
     ],
     "tags": [],
     "useCases": [],
-    "keywords": []
+    "keywords": [],
+    "dependencies": {
+        "CommonTypes": {
+            "mandatory": true,
+            "display_name": "Common Types"
+        }
+    },
+    "displayedImages": [
+        "CommonTypes"
+    ]
```

Comment:
Why did you do this? It's added automatically in the build

Reply:
i ran find-dependencies, and it generated it, want me to delete it?
@ShahafBenYakir